students/asotelo/lesson04/mailroom2.py

```diff
@@ -0,0 +1,111 @@
+#!/usr/bin/env python3
+
+import getpass
+username = getpass.getuser()
+
+'''
+Author: Alex Sotelo
+Lesson 4 Assignment 1
+Requirement: https://uwpce-pythoncert.github.io/PythonCertDevel/exercises/mailroom-part2.html
+'''
+
+donors = {
+    'Alex Sotelo': [175, 180], 'Juan Hirano': [200], 'Elon Musk': [12500, 1400, 1900], 'Akira Toriyama': [20, 80, 120],
+    'Charles Darwin': [10, 20], 'Gordon Ramsey': [100000]}
+
+def thank_you():
+    while True:
+        thanks_query = input(f"Welcome {username}! Either type 'list' to view the list of donors or your full name to "
+                             f"make a donation. \n").title()
+        if thanks_query == 'List':
+            print("Here is a list of our donors:")
+            for donor in donors:
+                print(donor)
+            break
+        elif thanks_query in ["Exit", "Cancel", "Quit", "Bye", "Stop"]:
+            return
+        elif len(thanks_query.split()) > 1:
+            for donor in donors:
+                if donor == thanks_query:
```

Comment:
since you're using a dictionary, there won't be a loop anymore. utilize a dictionary key look up here